packages/core/src/video/VideoForDevelopment.tsx

```diff
@@ -40,6 +32,7 @@ export const VideoForDevelopment: React.FC<RemotionVideoProps> = (props) => {
 
 		if (!playing || absoluteFrame === 0) {
 			videoRef.current.currentTime = currentTime;
+			videoRef.current.pause();
 		}
 
 		if (videoRef.current.paused && !videoRef.current.ended && playing) {
```

Comment:
I think the changes in this file are not necessary anymore right?

Reply:
Yup changes in this file is not necessary.